src/pages/general/navigation/NavigationStyles.tsx

```diff
@@ -94,14 +94,14 @@ export default class NavigationStyles {
 
 		.dropdown-menu {
 			position: absolute;
-			background-color: ${COLORS.PRIMARY} !important;
+			background-color: ${props => props.transparency ? 'transparent' : `${COLORS.PRIMARY}`} !important; //fix background later
```

Comment:
```suggestion
			background-color: ${props => props.transparency ? 'transparent' : `${COLORS.PRIMARY}`} !important; // TODO: fix background later
```
I like to add stuff like this as TODO comments